src/test/java/net/openhft/chronicle/queue/ChronicleQueueIndexTest.java

```diff
@@ -292,4 +293,157 @@ public void writeReadMetadata() {
             }
         }
     }
+
+    // https://github.com/OpenHFT/Chronicle-Queue/issues/822
+    private void driver0(String[] strings, boolean[] meta, long millis) {
+
+        assert (strings.length == meta.length);
+
+        File tmp = new File("/dev/shm/Issue882");
```

Comment:
not cross platform

Reply:
other than this minor change this all LGTM although I would not rely on just me ;)

Reply:
Thanks @JerryShea  - yes, only intended while testing locally. Swapped to getTmpDir()